test/shared/applications/javascript/base/build/domain/sampleContext/sampleAggregate/commands/authorize.js

```diff
@@ -13,11 +13,18 @@ const authorize = {
   },
 
   isAuthorized (state, command) {
-    return command.shouldAuthorize;
+    // Reject authorization if we cannot see the state properties.
+    // That allows to verify that `handleCommand` passes the state value
+    // instead of the `CurrentAggregateState` wrapper object.
+    if (!Object.prototype.hasOwnProperty.call(state, 'domainEventNames')) {
```

Comment:
Although it's not exactly the same, I think I'd prefer the more easily readable version of this without `hasOwnProperty`:

```suggestion
    if (!state.domainEventNames) {
```

Reply:
This should be fixed by now.